package.json

```diff
@@ -14,63 +14,61 @@
     "createUser": "node ./src/scripts/createUser.js"
   },
   "dependencies": {
-    "axios": "^0.15.3",
-    "babel-loader": "^6.2.10",
-    "babel-plugin-lodash": "^3.2.11",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-polyfill": "^6.22.0",
-    "babel-preset-env": "^1.6.0",
-    "babel-preset-react": "^6.16.0",
-    "bcryptjs": "^2.4.3",
+    "axios": "~0.16.2",
+    "babel-loader": "~7.1.2",
+    "babel-plugin-lodash": "~3.2.11",
+    "babel-plugin-transform-object-rest-spread": "~6.26.0",
+    "babel-polyfill": "~6.26.0",
+    "babel-preset-env": "~1.6.0",
+    "babel-preset-react": "~6.24.1",
+    "bcryptjs": "~2.4.3",
     "body-parser": "~1.15.1",
-    "chart.js": "^2.5.0",
-    "chartjs-plugin-datalabels": "^0.1.0",
+    "chart.js": "~2.7.0",
+    "chartjs-plugin-datalabels": "~0.1.0",
     "cookie-parser": "~1.4.3",
-    "cross-env": "^5.0.5",
-    "csv": "^1.1.0",
+    "cross-env": "~5.0.5",
+    "csv": "~1.1.1",
     "debug": "~2.2.0",
-    "dotenv": "^2.0.0",
-    "es6-promisify": "^5.0.0",
+    "dotenv": "~2.0.0",
+    "es6-promisify": "~5.0.0",
     "express": "~4.13.4",
-    "fs": "0.0.1-security",
-    "jsonwebtoken": "^7.2.1",
-    "lodash": "^4.17.4",
-    "lodash-webpack-plugin": "^0.11.0",
-    "moment": "^2.15.2",
+    "jsonwebtoken": "~7.2.1",
+    "lodash": "~4.17.4",
+    "lodash-webpack-plugin": "~0.11.4",
+    "moment": "^2.18.1",
```

Comment:
The latest version of moment is patched

Reply:
Also getting: 'Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged and will be removed in an upcoming major release. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.'